gax-httpjson/src/main/java/com/google/api/gax/httpjson/ProtoRestSerializer.java

```diff
@@ -134,10 +142,11 @@ public void putQueryParam(Map<String, List<String>> fields, String fieldName, Ob
   /**
    * Serializes a message to a request body in a form of JSON-encoded string.
    *
-   * @param fieldName a name of a request message field this message belongs to
    * @param fieldValue a field value to serialize
+   * @param numericEnum a boolean flag that determine if enum values should be serialized to number
+   *     or not
    */
-  public String toBody(String fieldName, RequestT fieldValue) {
-    return toJson(fieldValue);
+  public String toBody(RequestT fieldValue, boolean numericEnum) {
```

Comment:
This method is public. Is it OK to change the signature?

Reply:
Yeah, I mentioned it in the description as well. This is technically a breaking change but since REGAPIC is not GA yet, I think it might be fine. Need @vam-google  to confirm.

Reply:
Please make it non-breaking (do another method). It will save us a lot of headache, because it is very likely that older client will needs to be compiled/used with new gax and it is simple to do it here.

Reply:
Added a new method.